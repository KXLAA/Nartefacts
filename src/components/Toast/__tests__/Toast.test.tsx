import { render } from '@testing-library/react'

import { Toast } from '..'

describe('<Toast />', () => {
  it('should render the heading', () => {
    const { getByRole } = render(<Toast />)
    const heading = getByRole('heading', { name: /Toast/i })

    expect(heading).toBeInTheDocument()
  })
})
