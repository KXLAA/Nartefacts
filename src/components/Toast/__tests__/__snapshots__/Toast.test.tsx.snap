// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Toast /> should render the component correctly 1`] = `
.c0 {
  width: 100%;
  max-width: 400px;
  background-color: #202020;
  border: 1px solid #5A5A5A;
  border-radius: 4px;
  padding: 4px;
  cursor: pointer;
}

.c0 span {
  border-radius: 4px;
}

.c1 {
  color: #5A5A5A;
  font-size: 24px;
  text-align: center;
  font-weight: bold;
  padding-top: 4px;
}

<div
  class="c0"
  href="/saved"
>
  <span
    style="box-sizing: border-box; display: block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative;"
  >
    <span
      style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 100% 0px 0px 0px;"
    />
    <img
      alt="image uploaded by a user to generate color palettes"
      data-nimg="responsive"
      decoding="async"
      src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
      style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
    />
    <noscript />
  </span>
  <p
    class="c1"
  >
    Saved
  </p>
</div>
`;
